src/libs/ReportUtils.ts

```diff
@@ -3078,7 +3075,7 @@ function buildOptimisticWorkspaceChats(policyID, policyName) {
     };
 
     const adminsChatData = buildOptimisticChatReport(
-        [currentUserAccountID],
+        [currentUserAccountID ?? -1],
```

Comment:
Why this differ from `currentUserAccountID ? [currentUserAccountID] : []` above. Any specific reasons?